homeassistant/components/sensor/translations/en.json

```diff
@@ -33,8 +33,8 @@
     },
     "state": {
         "_": {
-            "off": "Off",
-            "on": "On"
+            "off": "[%key:common::state::off%]",
+            "on": "[%key:common::state::on%]"
         }
     },
     "title": "Sensor"
```

Comment:
This shouldn't be part of this PR.

Reply:
Thanks, not sure how that ended up in there. Fixed in f426c71